form-builder/src/test/java/com/dsc/form_builder/TextFieldProviders.kt

```diff
@@ -13,6 +13,34 @@ object EmailArgumentsProvider : ArgumentsProvider {
     )
 }
 
+object PhoneArgumentsProvider : ArgumentsProvider {
+    override fun provideArguments(context: ExtensionContext?): Stream<out Arguments> = Stream.of(
+        Arguments.of("test", false),
+        Arguments.of("88005553535", true),
+        Arguments.of("+8(800) 555-35-35", true),
+        Arguments.of("8-800-555-35-35-", false),
+    )
+}
+
+object WebUrlArgumentsProvider : ArgumentsProvider {
+    override fun provideArguments(context: ExtensionContext?): Stream<out Arguments> = Stream.of(
```

Comment:
Could you add a test for "www.test.com" It's a valid web url and should be factored in. The test is failing once we add this.

Reply:
thanks for the review! fixed the validator in the new commit